components/CompanyCard.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Company } from '../models/Company.model'
+
+const CompanyCard = ({ company }: { company: Company }): JSX.Element => {
+  return (
+    <div className="w-10/12 h-32 bg-gray-100 rounded shadow-xl flex items-center mb-8">
```

Comment:
Now I understand your question regarding the _w-10/12_ class. Actually, I never got to the point of using this kind of width setting with Tailwind. We've missed out to apply the classes in a mobile-first approach, which is probably the reason why this _w-10/12_ class is necessary at all.